### DISCUSSION TITLE
Can I make parameter into upstream uri ?

### DISCUSSION THREAD

**Discussion by gracekk** [apache/apisix#6573] [Q&A]
Can proxy-rewrite or some plugin can rewrite uri from request parameter?
e.g.
original request like https://abc.com/getuser?id=123
and I want query upstream uri like https://abc.com/123/getuser
and also some condition want query upstream like https://abc-upstream.com/getuser?userid=123
Any ideas？thx

**Comment by tzssangglass** [comment]
it's better to use `proxy-rewrite` to do this.
